bc_obps/service/contact_service_v2.py

```diff
@@ -48,17 +33,12 @@ def list_contacts_v2(
         return cast(QuerySet[Contact], queryset)
 
     @classmethod
-    def get_if_authorized_v2(cls, user_guid: UUID, contact_id: int) -> Optional[Contact]:
-        user = UserDataAccessService.get_by_guid(user_guid)
-        user_contacts = ContactDataAccessService.get_all_contacts_for_user(user)
-        contact = user_contacts.filter(id=contact_id).first()
-        if user.is_industry_user() and not contact:
-            raise Exception(UNAUTHORIZED_MESSAGE)
-        return contact
+    def get_by_id(cls, contact_id: int) -> Contact:
+        return Contact.objects.get(id=contact_id)
```

Comment:
We already have this in `ContactDataAccessService`.

Reply:
:facepalm: